src/services/kratos/errors.ts

```diff
@@ -0,0 +1,25 @@
+import { AuthenticationError } from "@domain/authentication/errors"
+import { ErrorLevel } from "@domain/shared"
+
+export class KratosError extends AuthenticationError {}
+export class LikelyNoUserWithThisPhoneExistError extends KratosError {}
+export class LikelyUserAlreadyExistError extends KratosError {}
+export class AuthenticationKratosError extends KratosError {}
+
+export class PhoneIdentityInexistentError extends KratosError {}
```

Comment:
All other errors already exist in `src/domain/authentication/errors.ts`. The only new error here is `PhoneIdentityInexistentError` and it should likely be added to that file and this file deleted